boltons/namedutils.py

```diff
@@ -1,5 +1,3 @@
-# -*- coding: utf-8 -*-
```

Comment:
worth mentioning somewhere: these were originally not just for python, but also emacs. But I believe since emacs 24 (~2012 I think), emacs assumes utf-8.